docs/source/serving/openai_compatible_server.md

```diff
@@ -35,14 +35,17 @@ vLLM also provides experimental support for OpenAI Vision API compatible inferen
 ## Extra Parameters
 vLLM supports a set of parameters that are not part of the OpenAI API.
 In order to use them, you can pass them as extra parameters in the OpenAI client.
-Or directly merge them into the JSON payload if you are using HTTP call directly.
+Or directly merge them into the HTTP headers or JSON payload if you are using HTTP call directly.
```

Comment:
It would be better to mention that only request ID can be set via HTTP headers.

Reply:
Please also revert this paragraph.

Reply:
Great, I'll add a new section for extra HTTP headers.